.ci.yaml

```diff
@@ -1616,7 +1616,7 @@ targets:
       task_name: web_size__compile_test
     scheduler: luci
 
-  - name: Linux android views
+  - name: Linux android_views
```

Comment:
Unfortunately, you can't change the name and remove bringup from this PR

Reply:
Ahh I'll leave the name as is then

Reply:
@CaseyHillers FYI, the ci.yaml validation check seems to be broken again.

Reply:
Thank you for letting me know! I reverted back to a safe cocoon roll out (temporarily). Feel free to push an empty commit to unblock you. After 5 PM PST I'm going to roll back out the bad version with extra debugging so I can fix the issue